entries/iteration.qmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "Time-constrained iterative workflow"
+date: last-modified
+---
+
+We take inspiration from some aspects of [Agile Software
+Development](https://agilemanifesto.org/) and
+[Scrum](https://www.scrum.org/resources/what-scrum-module), especially this idea
+of time-boxed periods ("iterations") of work that follow a specific pattern and process
+for incrementally building or improving things.  An iteration for us follows
+this general sequence of steps:
+
+- Team lead creates a GitHub Project Board for the upcoming iteration using the "Iteration" template.
+- The first meeting of the iteration is to discuss and plan the specifics of the iteration.
+    - Decide on the end output (the "increment" or "milestone").
+    - Brainstorm and write out as many tasks as possible that are needed to complete the increment or milestone, or select them from the current list. A task is a GitHub Issue.
```

Comment:
Maybe team members + team lead should add these before the first meeting, so we don't have to think about them "on the spot" but have more time to think through the steps/items needed? Then this first meeting could be about discussing the items already added to the board and "fine-tune" them?